src/client/components/App/index.css

```diff
@@ -0,0 +1,6 @@
+Iframe {
+    position: absolute;
+    width: 99%;
+    height: 100%;

```

Comment:
I made the width 99% because with 100%, the iframe had a tiny bit of scroll so it felt choppy.